content/learn.md

```diff
@@ -247,6 +247,28 @@ dasharo_faq:
 
         If you don’t have a solution yet, let Dasharo become your answer. [Contact us](/#get-a-quote)
 
+    - title: What kind of consulting services that 3mdeb can provide other than online courses?
+      content: >-
+         In addition to training, we provide in-house development and maintenance to keep hardware and software up to date or provide ad-hoc support to your development team.
+
+    - title: Where I can find your GitHub/issues?
+      content: >-
+        - Visit [github.com/Dasharo](https://github.com/Dasharo) where you can find details about our company and repositories.
+
+        - [Matrix Dasharo Space] is a place where you can find solutions to many problems and if not - our community will gladly help you. We encourage you to troubleshoot your open-source firmware issues there. Here is the invitation [Link](https://matrix.to/#/#dasharo:matrix.org).
+
+    - title: Can I have got Dasharo individually?
+      content: >-
+        Yes, you can get it four ways:
+        
+        - Join the newsletter with the latest firmware realease 
+
+        - Find firmware links products on our and client's websites
```

Comment:
@MateuszKochner add link from the 3rd point here, as a `on our` method, then remove 3rd method. Use also more detailed description: try to help where users can find the download links (client's websites with desired products which supports Dasharo firmware). Example from NovaCustom should be nice.